Manager/src/test/java/io/raspberrywallet/manager/ManagerTest.java

```diff
@@ -14,11 +15,13 @@
 class ManagerTest {
     private static Manager manager;
     static Bitcoin bitcoin;
+    static TemperatureMonitor temperatureMonitor;
 
     @BeforeAll
     static void setup() {
         bitcoin = Mockito.mock(Bitcoin.class);
-        manager = new Manager(bitcoin);
+        temperatureMonitor = Mockito.mock(TemperatureMonitor.class);
```

Comment:
Mocking the process would have more sense. The most volatile part is when `TemperatureMonitor` parses stdout of the process.

Reply:
You are right, but this is `ManagerTest`, where we test `Manager` behavior, not `TemperatureMonitor` implementation. We mock everything besides `Manager` in `ManagerTest`. If you want to test `TemperarureMonitor` then do it in `TemperatureMonitorTest` class